opensrp-chw-core/src/test/java/org/smartregister/chw/core/dao/FHIRBundleDaoTest.java

```diff
@@ -22,6 +26,7 @@
 import static org.smartregister.chw.core.utils.Utils.getRandomGeneratedId;
 
 @RunWith(PowerMockRunner.class)
+@Config(application = TestApplication.class, shadows = {ContextShadow.class})
```

Comment:
This is not required if test is being run with Powermock

Reply:
Done